### PR TITLE
add dataaction attributes to permissions and Role Definition calls

### PR DESCRIPTION
**Pull request by darshanhs90**
https://github.com/Azure/azure-rest-api-specs/pull/2356
https://github.com/Azure/azure-rest-api-specs/pull/2502

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Geneate your sdk using generate.cmd
Also build your generated sdk as
msbuild build.proj /t:build /p:Scope=SDKs\Authorization
This will generate two meta data files that are needed in this PR.

**Comment by darshanhs90** [comment]
@shahabhijeet have updated the metadata files

**Comment by shahabhijeet** [comment]
@darshanhs90 msbuild build.proj /t:Build /p:Scope=SDKs\Authorization
will generate a new meta data file *.props and this is needed as your API version has changed.

**Comment by darshanhs90** [comment]
@shahabhijeet Had forgot to push that changes,Added the props file

**Comment by darshanhs90** [comment]
@shahabhijeet Let me know if there is anything blocking for this PR

**Review by shahabhijeet — COMMENTED** [review]
@darshanhs90 after generation, please execute
msbuild build.proj /t:Build /p:Scope=SDKs\Authorization
this will create/update .props file (meta data file needed to document API versions used in SDK)

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please make the updates

File: src/SDKs/Authorization/AzSdk.RP.props [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
```
Comment (darshanhs90) [inline src/SDKs/Authorization/AzSdk.RP.props]: @shahabhijeet this it the required props file,correct?

**Comment by darshanhs90** [comment]
@shahabhijeet have included the props file in the earlier commit,let me know if that file isnt the one you had mentioned

**Comment by darshanhs90** [comment]
@shahabhijeet Let me know if there is anything blocking for this PR

File: src/SDKs/Authorization/AzSdk.RP.props (hunk 1 above)
Comment (shahabhijeet) [reply src/SDKs/Authorization/AzSdk.RP.props]: @darshanhs90 yes this is correct

File: src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Provides capabilities to query Microsoft Azure Management Authorization.</Description>
 		<AssemblyTitle>Authorization Library for Microsoft Azure Management Authorization</AssemblyTitle>
-		<Version>2.6.0-preview</Version>
+		<Version>2.7.0-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj]: @darshanhs90 missing < pacakgerelesenotes >

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj (hunk 2 above)
Comment (darshanhs90) [reply src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj]: @shahabhijeet  done,added it

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**